games-simulation/flightgear/patches/flightgear-2018.2.2.patchset

```diff
@@ -0,0 +1,68 @@
+Date: Sun Oct 14 12:59:05 GMT 2018
+From: Ken Mays <kmays2000@gmail.com>
+Subject: Haiku fixes for FlightGear 2018.2.2
+
+diff --git a/CMakeModules/DetectBrowser.cmake.orig b/CMakeModules/DetectBrowser.cmake
+index 6d4be54..bd94e56 100644
+--- a/CMakeModules/DetectBrowser.cmake.orig
++++ b/CMakeModules/DetectBrowser.cmake
+@@ -16,7 +16,7 @@ else()
+         endif()
+     else()
+         # Default for non Linux/non Mac/non Windows platform...
+-        set(WEB_BROWSER "firefox" CACHE STRING "Command to open web browser")
++	set(WEB_BROWSER "WebPositive" CACHE STRING "Command to open web browser")
```

Comment:
I think you can drop this patch and, instead, insert this line in `INSTALL` where cmake is invoked:
```diff
+		-DWEB_BROWSER:STRING="WebPositive" \
```